seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hive-e2e/src/test/java/org/apache/seatunnel/e2e/connector/hive/HiveIT.java

```diff
@@ -228,6 +228,8 @@ private void executeJob(TestContainer container, String job1, String job2)
     }
 
     @TestTemplate
+    @Disabled(
+            "[HDFS/COS/OSS/S3] is not available in CI, if you want to run this test, please set up your own environment in the test case file, hadoop_hive_conf_path_local and ip below}")
```

Comment:
cc @zhangshenghang

Reply:
@xiaochen-zhou  Hi, why disable? This CI can run.

Reply:
> @xiaochen-zhou Hi, why disable? This CI can run.

![image](https://github.com/user-attachments/assets/8621f7fd-34ba-4a73-8bd1-7ce83603deda)
 I encountered several CI failures, and I saw this comment in the code. Can this CI run?

Reply:
> > @xiaochen-zhou Hi, why disable? This CI can run.
> 
> ![image](https://private-user-images.githubusercontent.com/27844885/393713245-8621f7fd-34ba-4a73-8bd1-7ce83603deda.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3MjMwNDMsIm5iZiI6MTczMzcyMjc0MywicGF0aCI6Ii8yNzg0NDg4NS8zOTM3MTMyNDUtODYyMWY3ZmQtMzRiYS00YTczLThiZDEtN2NlODM2MDNkZWRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDA1MzkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2NTQ4NDdiNDQ4MTMzYTVlZWUyYWEzNjA3YjVjNDkxMzk4NzE4OWM1ZThiOTYxODgxMjBiNGJhMmRlOTJjZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mtw1PqitolzcX5RyeJkM4o3gNoVP3hzGrKwY6qWCbUs) I encountered several CI failures, and I saw this comment in the code. Can this CI run?

Yes, it is operational.  We can analyze the CI errors in detail.

Reply:
> > > @xiaochen-zhou Hi, why disable? This CI can run.
> > 
> > 
> > ![image](https://private-user-images.githubusercontent.com/27844885/393713245-8621f7fd-34ba-4a73-8bd1-7ce83603deda.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3MjMwNDMsIm5iZiI6MTczMzcyMjc0MywicGF0aCI6Ii8yNzg0NDg4NS8zOTM3MTMyNDUtODYyMWY3ZmQtMzRiYS00YTczLThiZDEtN2NlODM2MDNkZWRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDA1MzkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2NTQ4NDdiNDQ4MTMzYTVlZWUyYWEzNjA3YjVjNDkxMzk4NzE4OWM1ZThiOTYxODgxMjBiNGJhMmRlOTJjZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mtw1PqitolzcX5RyeJkM4o3gNoVP3hzGrKwY6qWCbUs) I encountered several CI failures, and I saw this comment in the code. Can this CI run?
> 
> Yes, it is operational. We can analyze the CI errors in detail.

Shouldn't HDFS be removed from this comment?

Reply:
> > > > @xiaochen-zhou Hi, why disable? This CI can run.
> > > 
> > > 
> > > ![image](https://private-user-images.githubusercontent.com/27844885/393713245-8621f7fd-34ba-4a73-8bd1-7ce83603deda.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3MjMwNDMsIm5iZiI6MTczMzcyMjc0MywicGF0aCI6Ii8yNzg0NDg4NS8zOTM3MTMyNDUtODYyMWY3ZmQtMzRiYS00YTczLThiZDEtN2NlODM2MDNkZWRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDA1MzkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2NTQ4NDdiNDQ4MTMzYTVlZWUyYWEzNjA3YjVjNDkxMzk4NzE4OWM1ZThiOTYxODgxMjBiNGJhMmRlOTJjZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mtw1PqitolzcX5RyeJkM4o3gNoVP3hzGrKwY6qWCbUs) I encountered several CI failures, and I saw this comment in the code. Can this CI run?
> > 
> > 
> > Yes, it is operational. We can analyze the CI errors in detail.
> 
> Shouldn't HDFS be removed from this comment?

@Hisoka-X There is a problem here, the underlying storage is not actually running on HDFS storage. Do you need to adjust the description name of the testing method, otherwise it may cause misunderstanding.

Reply:
> There is a problem here, the underlying storage is not actually running on HDFS storage. Do you need to adjust the description name of the testing method, otherwise it may cause misunderstanding.

+1. Please go ahead.